src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -13,6 +13,31 @@
                     #{studyId}
                 </foreach>
             </if>
+            <!-- sample-lists-counts filter -->
+            <if test="studyViewFilter.caseLists != null and !studyViewFilter.caseLists.isEmpty()">
+                INTERSECT
+                <!-- get samples that are in each sample list in the filter -->
```

Comment:
lets word it using the filter concept:  "Filter for samples which belong to the sample lists (aka case lists) selected by the user and appearing in the studyViewFilter.caseLists collection".   And lets put up above at line 16